styles.css

```diff
@@ -0,0 +1,1240 @@
+
+}
+
+#tweak-head5 {
+    grid-area: head;
+    display: inline;
+    background-color: #a2a2a2;
+    margin-bottom: 980%;
+    margin-top: 10%;
+    margin-left: 14%;
+    margin-right: 10%;
+    border-radius: 60%;
+}
+
+#tweak-head5 img {
+    object-fit: cover;
+    height: 100%;
+    width: 100%;
+    border-radius: 60%;
+}
+
+#tweak-wings5 {
+    grid-area: header;
+    padding-top: 2mm;
+}
+
+#tweak-body5 {
+    grid-area: body;
+    padding-top: 0.5mm;
+}
+
+#tweak-body5-pic {
+    margin-top: 2mm;
+    grid-area: pic;
+}
+
+#tweak-body5-pic img {
+    width: 97%;
+    height: 600px;
+    border-radius: 5%;
+}
+
+#tweak-tail5 {
+    grid-area: dots;
+    padding-top: 2mm;
+}
+
+#tweak-branch5 {
+    grid-area: branch;
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+    max-width: 490px;
+}
+
+#tweak-branch5 p {
+    font-weight: bold;
+    font-size: 3mm;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #a2a2a2;
+
+}
+
+#reply5,
+#number-reply5,
+#retweak5,
+#number-retweaks5,
+#like5,
+#number-likes5,
+#share5 {
+    width: 23px;
+    margin-top: 3mm;
+
+}
+
+#number-reply5,
+#number-retweaks5,
+#number-likes5 {
+    margin-top: 4mm;
+    justify-items: left;
+    margin-right: 2.6cm;
+
+}
+
+#tweak-body1 span:hover {
+    text-decoration: underline;
+}
+
+#tweak-body4 span:hover {
+    text-decoration: underline;
+}
+
+#tweak-body5 span:hover {
+    text-decoration: underline;
+}
+
+.bottom-tweak:hover {
+    text-decoration: underline;
+}
+
+.tweak1:hover {
+    background-color: rgb(247 247 247);
+}
+
+.tweak2:hover {
+    background-color: rgb(247 247 247);
+}
+
+.tweak3:hover {
+    background-color: rgb(247 247 247);
+}
+
+.tweak4:hover {
+    background-color: rgb(247 247 247);
+}
+
+.tweak5:hover {
+    background-color: rgb(247 247 247);
+}
+
+.bottom-tweak:hover {
+    background-color: rgb(247 247 247);
+}
+
+
+.bottom-tweak {
+    padding-top: 33px;
+    padding-left: 33px;
+    color: rgb(29 156 240);
+    padding-bottom: 10px;
+    text-align: center;
+    font-weight: bold;
+    font-size: medium;
+    font-family: Arial, Helvetica, sans-serif;
+    border-left: 2px solid rgb(204 215 221);
+    border-right: 2px solid rgb(204 215 221);
+}
+
+.right-header {
+    grid-area: right-header;
+    background-color: #e7e7e7;
+    margin: 2%;
+    display: grid;
+    grid-template-columns: 1fr 10fr;
+    padding-top: 2mm;
+    padding-left: 7mm;
+    border-radius: 25px;
+    gap: 2px;
+}
+
+.right-header p {
+    font-weight: 550;
+    font-size: small;
+    font-family: Arial, Helvetica, sans-serif;
+    color: rgb(102 118 130);
+    padding-top: 0.5mm;
+
+
+}
+
+.right-header img {
+    width: 20px;
+    display: inline-block;
+}
+
+.main-right {
+    grid-area: main-right;
+    min-width: 200px;
+    border-radius: 25px;
+    margin-top: 2mm;
+    margin-right: 2mm;
+    margin-bottom: 2mm;
+    margin-left: 2mm;
+    background-color: rgb(247 249 249);
+    );
```

Comment:
This is causing an error when trying to view the submission in code sandbox. You might find the solution is working on your machine (browsers can sometimes ignore errors like these and still render the page) but a syntax error like this would cause your webpage to not render when we come to the later modules, so just be careful with it! If visual studio code is showing a filename in red it means there is an error in the file. Typically you'll be able to see where the error is on the right hand side, it should show a small red line next to the scroll bar